api/holiday_acres_api/serializers.py

```diff
@@ -50,6 +50,35 @@ def update(self, horse, data):
         # the old instance and creating a new instance
         if "feed" in data.keys():
 
+            # validate feed amount, feed type, and unit
+            for feed in data["feed"]:
+                if feed["amount"] <= 0:
+                    print("Must be a positive value")
+                    return horse
+                if feed["amount"] >= 10:
+                    print("No Horse eats that much")
+                    return horse
```

Comment:
To answer your question about printing/what to return, I'd recommend returning a `500` response like [we did here with register account validations](https://github.com/mstephenhunt/holiday_acres/blob/29ea37a63cbc14300a3b9d3d267fb873f7a9d60f/api/holiday_acres_api/views.py#L69-L71)